info.yml

```diff
@@ -92,4 +92,4 @@ requires_engine_version:
 # the frameworks required to run this app
 frameworks:
     - {"name": "tk-framework-shotgunutils", "version": "v5.x.x", "minimum_version": "v5.2.0"}
-    - {"name": "tk-framework-qtwidgets", "version": "v2.x.x", "minimum_version": "v2.10.5"}
+    - {"name": "tk-framework-qtwidgets", "version": "v2.x.x", "minimum_version": "v2.10.9"}
```

Comment:
Ok so we need to make sure we release tk-framework-qtwidgets v2.10.9 before merging this PR, right?

Reply:
yup!
I've released this framework this morning precisely to create this PR :)